### ISSUE TITLE
3.0.0 nuget package still targeting .Net 3.5 framework

### ISSUE THREAD

**Issue by bryanrengo** [segmentio/Analytics.NET#52]
Hey, thanks for posting a new version on nuget!

I have been trying to use this package for a netcore1.0 library and the only way I currently can do it is to clone this repo and pull the Analytics project directly into my solution. For build to production sake, I'd like to pull in the nuget package instead.

When I try to reference the 3.0.0 nuget package, it throws this error:

```
Restoring packages for xxx.csproj...
  GET http://nuget/nuget/FindPackagesById()?id='Analytics'
  GET https://api.nuget.org/v3-flatcontainer/analytics/index.json
  OK http://nuget/nuget/FindPackagesById()?id='Analytics' 6ms
  OK https://api.nuget.org/v3-flatcontainer/analytics/index.json 284ms
  GET https://api.nuget.org/v3-flatcontainer/analytics/3.0.0/analytics.3.0.0.nupkg
  OK https://api.nuget.org/v3-flatcontainer/analytics/3.0.0/analytics.3.0.0.nupkg 193ms
Installing Analytics 3.0.0.
Package Analytics 3.0.0 is not compatible with netcoreapp1.0 (.NETCoreApp,Version=v1.0). Package Analytics 3.0.0 supports: net (.NETFramework,Version=v0.0)
Package restore failed. Rolling back package changes for 'xxx'.
Time Elapsed: 00:00:01.0880009

```
It appears the project config/csproj file is correct at targeting the netstandard1.3, but the nuspec file needs to be updated to allow for the newer frameworks to be targeted.

**Comment by f2prateek** [comment]
@bryanrengo gotcha, thanks for letting us know. I'm not totally familiar with what it would take to get nuget  + netcore to work. I'd have to look into this a bit more.

If you happen to know the changes required, we'd be happy to accept a PR!

@spewu helped us out with the netstandard support, and might have some pointers as well.

**Comment by spewu** [comment]
@f2prateek I am not sure how you currently build the NuGet packages, but I would suggest you just use the dotnet CLI. It has a `pack` command for creating a NuGet package: https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-pack

If you are currently building the NuGet packages manually, you could also consider creating an account on e.g. https://www.appveyor.com - it is free for open source projects.

This blog post has a pretty good walk-through in case you are interested in going that way: https://andrewlock.net/publishing-your-first-nuget-package-with-appveyor-and-myget/

**Comment by f2prateek** [comment]
Ah neat, we are using appveyor for the C++ library, that might be a good option here too. RIght now the process is pretty manual https://github.com/segmentio/Analytics.NET/blob/master/RELEASING.md.

Either way though, for @bryanrengo's case, it sounds like we'd need to update the https://github.com/segmentio/Analytics.NET/blob/master/Analytics.nuspec to allow for newer frameworks to be targetted.

**Comment by spewu** [comment]
@f2prateek As far as I know, the nuspec file is not needed anymore actually ... maybe I should have deleted it in my pull request :)

**Comment by f2prateek** [comment]
@spewu we'd still need the nuspec file to upload to nuget I think, no?

**Comment by f2prateek** [comment]
This should be fixed as of 3.1.0-alpha. Please try it out, and let me know if you run into any issues!
